### ISSUE TITLE
Crashes on DT packages

### ISSUE THREAD

**Issue by andrewbranch** [arethetypeswrong/arethetypeswrong.github.io#113]
- When the implementation package is [missing from the registry](https://jakebailey.dev/dt-dashboard/), the error message is bad (cannot convert null or undefined to object)
- These threw `Cannot read properties of undefined (reading 'filename')`:
  - is
  - openurl
  - youtube
- There was one instance of #89
- These threw `fetch error` (probably transient network issue):
  - emartech__cls-adapter
  - ember-cli-fastboot
  - ember-mocha
  - ember-changeset-validations
  - switchery
  - swiz
  - sx127x-driver
  - sybase-promised
  - sylvester
  - sylvester-es6
  - symlink-or-copy
  - symbol-tree
  - symphony-api-client-node
  - sync-fetch
  - synaptic

All errors:

<details><summary>Expand</summary>

```
3box
acc-wizard
add2home
adeira__test-utils
akumina-core
akumina-core/v4
ali-oss
allure-js-commons
amazon-cognito-auth-js
amcharts
amplifier
amplify
amplify-deferred
angular
angular-agility
angular-animate
angular-aria
angular-bootstrap-calendar
angular-cookies
angular-dialog-service
angular-httpi
angular-idle
angular-material
angular-meteor
angular-notifications
angular-notify
angular-odata-resources
angular-q-spread
angular-resource
angular-route
angular-sanitize
angular-scenario
angular-ui-notification
angular-ui-router
angular-ui-tree
angular.throttle
angularfire
angularlocalstorage
ansicolors
aos
apigee-access
app-root-dir
appframework
applepayjs
appletvjs
aqb
arbiter
arcgis-to-geojson-utils
argv
artillery
assertsharp
async-cache
async-polling
async-retry
atlaskit__layer
atmosphere.js
atpl
audio-context
auth0.widget
aws-lambda
axe-webdriverjs
azdata
azure-mobile-services-client
azure-sb
babel-types
babylon-walk
backbone.localstorage
backgrid
better-curry
bgiframe
bigint
bignum
bit-array
bitcoin-computer__lib
bitcoin-computer-bitcore
blissfuljs
bluebird-global
boolify-string
boom
boom/v3
boom/v4
bootstrap-colorpicker
bootstrap-datepicker
bootstrap-filestyle
bootstrap-maxlength
bootstrap-switch
bootstrap-touchspin
bootstrap-treeview
bootstrap.paginator
bootstrap.timepicker
bootstrap.v3.datetimepicker/v3
box2d
breeze
bunyan-blackhole
calq
cannon
canvasjs
carbon__layout
carbon__motion
carbon__themes
carbon__type
catbox
catbox/v7
chai-datetime
chartjs-plugin-doughnutlabel-rebourne
chessboardjs
chocolatechipjs
chrome-apps
chui
closure-compiler
cls-hooked
code
coinbase
colab
color/v0
com.wikitude.phonegap.wikitudeplugin
commangular
compose-function
connect-flash
consolidate
content-type
contextjs
cordova-ionic
cordova-plugin-background-mode
cordova-plugin-email-composer
cordova-plugin-ibeacon
cordova-plugin-mapsforge
cordova-plugin-ms-adal
core-js
countdown
coverup
cradle
createjs-lib
crossfilter
crossroads
crumb
cryptiles
cryptojs
css
css-font-loading-module
css-generator
css-modules
csurf
cybozulabs-md5
d3-box
d3-tip
d3.slider
data-driven
db-migrate-base
db-migrate-pg
debug
decorum
deep-freeze
deku
deoxxa-content-type
deployjava
desmos
devexpress-web
devexpress-web/v161
devexpress-web/v162
devexpress-web/v171
devexpress-web/v172
devexpress-web/v181
devexpress-web/v182
devexpress-web/v191
devexpress-web/v192
devexpress-web/v201
df-visible
dhtmlxgantt
dhtmlxscheduler
doccookies
dock-spawn
documentdb
documentdb-server
documentdb-session
domurl
double-ended-queue
dropboxjs
dts-bundle
duosecurity__duo_web
durandal/v1
dvtng-jss
dw-bxslider-4
dymo-label-framework
dynatable
dynatrace
easy-api-request
easy-jsend
easy-xapi
easy-xapi-utils
easydate
ebongarde-root
egg.js
ejs-locals
elasticsearch
email-templates
emartech__cls-adapter
ember
ember-changeset-validations
ember-cli-fastboot
ember-mocha
ember-test-helpers
ember-test-helpers/v0
ember-testing-helpers
ember/v1
ember/v1
ember/v2
ember/v2
ember/v3
ember/v3
emissary
emscripten
engine-check
epub
eq.js
es6-collections
es6-weak-map
esbuild-plugin-import-map
espruino
eth-sig-util
event-to-promise
express-pino-logger
express-serve-static-core
express-simple-locale
express-wechat-access
extjs
extract-text-webpack-plugin
facebook-js-sdk
facebook-pixel
factory-girl-objection-adapter
farbtastic
fast-levenshtein
favico.js
featherlight
ffi
ffmpeg
ffmpeg__libav-core
ffmpeg-static
ffmpeg.js
fibjs
figma
filesystem
filewriter
finch
flight
flipsnap
flot
flowjs
fm-websync
fontoxml
formidable/v1
foundation
friendly-errors-webpack-plugin
ftdomdelegate
gae.channel.api
gamequery
gandi-livedns
gapi.analytics
gapi.calendar
gapi.drive
gapi.pagespeedonline
gapi.people
gapi.plus
gapi.translate
gapi.urlshortener
gapi.youtube
gapi.youtubeanalytics
gaugejs
generic-functions
gensync
gently
geometry-dom
git
gldatepicker
glidejs
glue
glue/v4
go
google__maps
google-ads-scripts
google-closure-compiler
google-drive-realtime-api
google-earth
google-libphonenumber
google-one-tap
google.analytics
google.feeds
google.geolocation
google.visualization
googlemaps.infobubble
gorilla-engine
graphql-resolve-batch
greasemonkey
greasemonkey/v3
grecaptcha
grecaptcha/v0
guid
gulp-angular-templatecache
gulp-changed
gulp-cheerio
gulp-coffeeify
gulp-dtsm
gulp-espower
gulp-help-doc
gulp-jasmine-browser
gulp-load-plugins
gulp-minify-css
gulp-mocha
gulp-ng-annotate
gulp-remember
gulp-sourcemaps
gulp-task-listing
gulp-util
h2o2
hapi
hapi__catbox
hapi__catbox-memory
hapi__joi
hapi__shot
hapi-auth-basic
hapi-auth-cookie
hapi/v12
hapi/v15
hapi/v16
hapi/v17
hapi/v8
hard-source-webpack-plugin
hasher
hashset
hashtable
hasura
headroom
hellosign-sdk
heroku-logger
hexo-bunyan
highlightjs
hmscore__react-native-hms-push
hoek
hookrouter
html-pdf
html-webpack-plugin
htmltojsx
hubspot-pace
hyperscript
hypertext-application-language
i18next-node-fs-backend
ibm-mobilefirst
iltorb
imagemagick
imagemapster
imgur-rest-api
inert
inert/v4
inherits
iniparser
intercomjs
ion-rangeslider/v1
ipaiva
iron
is
iscroll
iscroll/v4
isomorphic-fetch
istanbul
istanbul-middleware
ix.js
jade
jake
jasmine-jquery
jasmine/v1
java-applet
javascript-astar
javascript-bignum
jdataview
jee-jsf
jfp
jpm
jqgrid
jquery-alertable
jquery-backstretch
jquery-cropbox
jquery-deparam
jquery-fullscreen
jquery-galleria
jquery-handsontable
jquery-jcrop
jquery-timeentry
jquery-truncate-html
jquery-urlparam
jquery.address
jquery.are-you-sure
jquery.autosize
jquery.base64
jquery.bbq
jquery.blockui
jquery.bootstrap.wizard
jquery.cleditor
jquery.clientsidelogging
jquery.color
jquery.colorbox
jquery.colorpicker
jquery.contextmenu
jquery.customselect
jquery.cycle
jquery.dropotron
jquery.dynatree
jquery.elang
jquery.fancytree
jquery.fileupload
jquery.filtertable
jquery.finger
jquery.flagstrap
jquery.form
jquery.fullscreen
jquery.gridster
jquery.highlight-bartaz
jquery.jnotify
jquery.joyride
jquery.jsignature
jquery.leanmodal
jquery.livestampjs
jquery.menuaim
jquery.mmenu
jquery.notifybar
jquery.noty
jquery.pjax
jquery.placeholder
jquery.pnotify
jquery.postmessage
jquery.prettyphoto
jquery.qrcode
jquery.rowgrid
jquery.simplemodal
jquery.simplepagination
jquery.simulate
jquery.sortelements
jquery.superlink
jquery.tagsmanager
jquery.tile
jquery.timeago
jquery.timepicker
jquery.timer
jquery.tinycarousel
jquery.tinyscrollbar
jquery.tipsy
jquery.total-storage
jquery.ui.datetimepicker
jquery.ui.layout
jquery.uniform
jquery.watermark
jquery.window
jquery/v1
jquery/v2
jquerymobile
jqueryui
js-clipper
js-schema
js-url
js.spec
jsbn
jsdeferred
jsfl
jsgraph
json-editor
json-merge-patch
json-patch
json-pointer
json-schema
json2csv
jsoneditoronline
jsonstream
jstorage
jsum
justifiedgallery
jwplayer
jxon
k6
kahoot.js-updated
kap-plugin
kendo-ui
kik-browser
kineticjs
knockback
knockout-amd-helpers
knockout-postbox
knockout-secure-binding
knockout.deferred.updates
knockout.editables
knockout.es5
knockout.kogrid
knockout.mapper
knockout.mapping
knockout.projections
knockout.punches
knockout.rx
knockout.validation
knockout.viewmodel
knockstrap
koa-csrf
koa-multer
koa-proxy
koa-router
koji-tools
kolite
konami.js
kos-core
kube-probe
lab
later
layui-src
lazypipe
leadfoot
leaflet-curve
leaflet-editable/v0
leaflet-gpx
leaflet-label
leaflet.awesome-markers/v0
leaflet.pm
leapmotionts
ledgerhq__hw-transport-u2f
lestate
level-sublevel
lightpick
line-reader
lls
lodash.isarray
lodash.now
logg
loopback-boot
lowlight
macaca-circular-json
magic-number
maildev
mailgun-js
mandrill-api
mapbox
mapbox__s3urls
mapsjs
markdown-it-lazy-headers
markitup
maskedinput
material-ui
mathjax
mcrypt
mcustomscrollbar
mdast
mdast/v3
mem-cache
merge-stream
messenger
meteor-accounts-phone
meteor-jboulhous-dev
meteor-persistent-session
meteor-prime8consulting-oauth2
meteor-publish-composite
meteor-roles
meteor-sjobs
method-override
microservice-utilities
microsoft-ajax
microsoft-live-connect
microsoft-sdk-soap
milliseconds
mimos
mina
minapp-env
mixpanel
mixto
mochaccino
mojang-gametest
mojang-minecraft
mojang-minecraft-server-admin
mojang-minecraft-ui
mojang-net
moment-business
moment-holiday
moment-precise-range-plugin
moment-shortformat
moment-strftime2
mongoose-promise
mongoose-sequence
msgpack
mu2
multer
multiparty
mumath
musickit-js
natural-sort
naver-whale
nedb-logger
neo4j
nes
netlify-auth-providers
ng-command
ng-facebook
ng-flow
ng-grid
ng-i18next
ngbootbox
ngprogress-lite
node_redis
node-fibers
node-mysql-wrapper
node-sass-middleware
node-uuid
node-xmpp-client
node-xmpp-core
nodegit
nodemailer/v3
nodeunit
noisejs
nomnom
noteflight-client
notify
notifyjs-browser
npmcli__ci-detect
nunjucks-date
nw.gui
nw.js
oauth.js
oblo-util
office-js
office-js-preview
oidc-token-manager
onflow__flow-js-testing
openjscad
openlayers
openlayers/v2
openlayers/v3
openssi-websdk
openui5
openurl
optimist
oracledb/v3
paralleljs
parcel-bundler
parcel-env
parse-torrent-file
passport
passport-github2
passport-saml
passport-strategy
passport-vkontakte
passport.socketio
pathfinding
paypal-cordova-plugin
payu-emea-sdk
pebblekitjs
phantom
phonegap
phonegap-facebook-plugin
phonegap-nfc
phonegap-plugin-barcodescanner
photoshop
pi-spi
picturefill
pino-multi-stream
pinterest-sdk
piwik-tracker
places
playerframework
playmusic
plupload
pngjs2
podium
polyline
polymer
popcorn
post-message-stream
postlight__mercury-parser
pouchdb-http
preact-i18n
precise
preloadjs
prismic-dom
proclaim
progressjs
project-name-generator
projections
promise-pg
promise-pool
promise-sftp
pump
purifycss-webpack
purl
q-retry
quill
qunit/v1
radius
random-string
rangy
raspi-pwm
raty
raven
raygun4js
rbac-a
rdf-data-model
rdfjs__serializer-jsonld-ext
react-better-password
react-cache
react-color
react-date-range/v0
react-document-meta
react-fa
react-facebook-login-component
react-geosuggest
react-infinite
react-input-mask
react-instantsearch-dom
react-instantsearch-native
react-jsonschema-form
react-key-handler
react-leaflet-markercluster
react-leaflet-markercluster/v2
react-linkify
react-native-bluetooth-serial
react-native-dotenv
react-native-google-signin
react-native-huawei-protected-apps
react-native-material-ui
react-native-modern-datepicker
react-native-orientation
react-native-toast-native
react-native/v0.63
react-native/v0.64
react-native/v0.65
react-native/v0.66
react-native/v0.67
react-native/v0.68
react-onsenui
react-pie-menu
react-pointable
react-router-navigation
react-router-navigation-core
react-router-redux
react-s-alert
react-scrollspy
react-sortable-tree
react-sortable-tree-theme-file-explorer
react-swf
react-tap-event-plugin
react-touch
react-user-tour
react-virtual-keyboard
read-package-tree
readmore-js
recharts-scale
redux-action-utils
redux-form/v7
redux-localstorage
remotedev-serialize
request
request-promise
request-promise-native
restify-plugins
rickshaw
riderize__passport-strava-oauth2
riot-api-nodejs
riot-games-api
riotjs
rison
rn-fetch-blob
rollup-plugin-buble
rollup-plugin-json
rollup-plugin-svelte-svg
rollup-plugin-url
rosie
route-parser
routie
royalslider
run-sequence
rx-jquery
rx-node
s3-uploader
safari-extension
safari-extension-content
sammy
sane
sanitize-s3-objectkey
sat
satnav
screeps-arena
scriptjs
scroller
scrollreveal
scrolltofixed
segment-analytics
sencha_touch
sequelize
sequelize/v3
sequencify
servicenow
set-tz
sharepoint
short-css-vars
shortid
shot
shot/v3
sigmajs
signalfx-collect
signalr/v1
simple-cw-node
simple-oauth2/v1
simple-oauth2/v2
simple-react-lightbox
simple-url-cache
simple-xml
sinon
sipml
six-runtime
ski
sleep
smart-fox-server
snekfetch
snowpack-env
sntp
socket.io.users
soundjs
space-pen
spectrum
sphere-engine-browser
sphere-engine-browser/v1
sprintf
standard-version
statsd-client
steam
storage-helper
stream-meter
stream-series
stream-to-array/v0
string-similarity
string-similarity/v1
stripe-v2
stripe-v3
strophe
styled-components-react-native
succinct
suitescript
superagent
supertest-as-promised
svg-injector
svg-maps__common
svgjs.draggable
svgjs.resize
sw-precache
swagger-express-middleware
swagger-schema-official
swig
swipe
swipeview
switchery
swiz
sx127x-driver
sybase-promised
sylvester
sylvester-es6
symbol-tree
symlink-or-copy
symphony-api-client-node
synaptic
sync-fetch
tableau
tea-merge
teechart
tether-shepherd
text-encoding
tile-reduce
timelinejs
timezone-js
tinajs__tina-redux
to-markdown
to-title-case-gouch
tooltipster
topo
topojson
torrent-stream
trayballoon
trim
twitch-browser
twitter-for-web
tz-format
uglify-es
umbraco
underscore-ko
underscore.string
unity-webapi
unzalgo
uploadcare
urix
url-join/v0
url-regex-safe
url-search-params
urlparser
urlrouter
usage
use-sync-external-store
user-event
user-idle-observer
utils-merge
uuid-browser
uuid/v2
uuid/v3
uws
valdr-message
vanilla-modal
vast-client
vast-client/v2
venn
vertx3-eventbus-client
viewporter
vimeo
vinyl-paths
vis
vision
vision/v4
viz.js
vmap
vortex-web-client
vscode
vscode-windows-registry
vue-datetime
w3c-hr-time
waitme
wallabyjs
wampy
weapp-api
webassembly-web-api
webcl
webcrypto
webgl-ext
webgl2
webpack-env
webrtc
websql
webvr-api
wepy-redux
why-did-you-update
wiiu
windows-mutex
winjs/v1
winjs/v2
winrt
winrt-uwp
winston-dynamodb
wonder.js
wordpress__custom-templated-path-webpack-plugin
wordpress__library-export-default-webpack-plugin
workerb-api
wreck
wreck/v7
wrench
xmldom
xmltojson
xmpp__jid
xrm
xrm/v6
xrm/v7
xrm/v8
xsockets
xss-filters
yandex-money-sdk
yar
ydn-db
youtube
youtube-dl
zeit__next-source-maps
zeit__next-typescript
zengin-code
zip.js
zmq
```
</details>

**Comment by bradzacher** [comment]
https://arethetypeswrong.github.io/?p=@types/yargs@17.0.32

```
Uncaught (in promise) Error: 'resolveImplementationPackageForTypesPackage' expects an @types package name and version
    at Jht (worker-eb03fba9.js:400:14792)
    at oOe (worker-eb03fba9.js:400:14392)
    at onmessage (worker-eb03fba9.js:400:17428)
```

**Comment by andrewbranch** [comment]
@bradzacher fixed in #133
